setup.py

```diff
@@ -45,7 +45,7 @@
 #
 
 PACKAGE_NAME = "liota"
-PACKAGE_VERSION = "0.3.1"
+PACKAGE_VERSION = "0.4"
```

Comment:
Having duplicate code and keeping on updating it in both places is not a good development practice. Keep only one setup.py and install liota-lite using an install parameter.

Reply:
As per the guidance of John Hawley we have designed two setup.py for Liota and Liota-lite as both have different files to bundle in the Liota package. The feature request can be marked for the common setup.py and taken up in next release. Changing the approach now will require changes in Pulse installer before the RTM, which is not advised.

Reply:
Sure, makes sense to delay it for now. Please raise a feature request to merge the two files.

Reply:
Sure can you approve the PR?